state/state_store.go

```diff
@@ -5,8 +5,8 @@
 
 package state
 
-// StateStore is an interface to perform operations on store
-type StateStore interface {
+// Store is an interface to perform operations on store
+type Store interface {
```

Comment:
rename the interface as package is already `state` (e.g. to avoid state.StateStore from consumer point of view)